mcs/tools/mkbundle/mkbundle.cs

```diff
@@ -1126,10 +1126,10 @@ static void LoadLocalizedAssemblies (List<string> assemblies)
 	
 	static readonly Universe universe = new Universe ();
 	static readonly Dictionary<string, string> loaded_assemblies = new Dictionary<string, string> ();
-	static readonly string resourcePathSeparator = (Path.DirectorySeparatorChar == '\\') ? $"\\{Path.DirectorySeparatorChar}" : $"{Path.DirectorySeparatorChar}";
 
 	public static string GetAssemblyName (string path)
 	{
+		string resourcePathSeparator = style == "windows" ? "\\" : "/";
```

Comment:
Doesn't this break https://github.com/grendello/mono/commit/21dec720833d6cb4b6f13cf604ca1c3d1b811dc9 ? i.e. you now embed `\` instead of `\\` like the old code on Windows.